docs/configuration_options.md

```diff
@@ -0,0 +1,303 @@
+tracked by [rollbar-agent](https://github.com/rollbar/rollbar-agent).
+
+## framework
+
+**Default** 'Plain'
+
+Indicates which framework you're using. Common options include 'Rails',
+'Sinatra', and 'Rack' to name a few.
+
+## ignored_person_ids
+
+**Default** `[]`
+
+Ids of people whose reports you wish to ignore. Only works in conjunction with a
+properly defined `person_method` or `person_id_method`.
+
+## logger
+
+The logger to use *instead of* the default logger. Especially useful in `scope`s
+where you wish to send log messages elsewhere.
+
+## payload_options
+
+Extra data to send with the payload. Use `scope` with `payload_options` to set
+specific payload options before sending.
+
+## person_method
+
+Rails only: A string or symbol giving the name of the method on the controller.
+Should return an object with an `id` method, and optionally `username` and
+`email` methods.
+
+If not using Rails:
+
+Populate the Rack `person_data` key with a hash containing `:id`, and optionally
+`:username` and `:email`.
+
+## person_id_method
+
+Rails only: a string or symbol giving the name of the method on the controller
+which returns the current user's id (a string). Ignored if `person_method`
+present.
+
+## person_username_method
+
+Rails only: a string or symbol giving the name of the method on the controller
+which returns the current user's username. Ignored if `person_id_method` not
+present. Ignored if `person_method` present.
+
+## person_email_method
+
+Rails only: a string or symbol giving the name of the method on the controller
+which returns the current user's email address. Ignored if `person_id_method`
+not present. Ignored if `person_method` present.
+
+## populate_empty_backtraces
+
+If you report a `new` exception, but do not `raise` it, the backtraces in Rollbar
+will be empty. Set `populate_empty_backtraces` to `true` to have Rollbar load
+the traces before sending them.
+
+## report_dj_data
+
+**Default** `true`
+
+Set to `false` to skip automatic reporting of Delayed job data. This can be
+handy if you manually report to Rollbar, or you have another way of catching and
+reporting those errors.
+
+## request_timeout
+
+**Default** `3`
+
+Set the request timeout for sending POST data to Rollbar.
+
+## root
+
+Set the server root, all stack frames outside that root are considered
+'non-project' frames. Also used to setup Github linking.
+
+## safely
+
+**Default** `false`
+
+When `true` evaluates `custom_data_method` in a `begin`/`rescue` block, returning
+`{}` if an error occurs. Otherwise reports the error to Rollbar.
+
+## scrub_fields
+
+Fields to scrub out of the parsed request data. Will scrub from `GET`, `POST`,
+url, and several other locations. Does not currently recurse into the full
+payload.
+
+## scrub_user
+
+**Default** `true`
+
+TODO: CORRECT THIS
+
+When true filters the user out of the URL.
+
+## scrub_password
+
+TODO: CORRECT THIS
+
```

Comment:
These values should be boolean and the purpose is what you have already there.

When scrubbing the URL, we scrub or not the user/password. By default true.

So, scrubbing URLs, we always scrub the query string, and the user and password is configurable. We'll scrub by default those.